src/axom/slic/docs/sphinx/sections/wrapping_slic_in_macros.rst

```diff
@@ -0,0 +1,78 @@
+.. ## Copyright (c) 2017-2019, Lawrence Livermore National Security, LLC and
+.. ## other Axom Project Developers. See the top-level COPYRIGHT file for details.
+.. ##
+.. ## SPDX-License-Identifier: (BSD-3-Clause)
+
+.. _sections/wrapping_slic_in_macros:
+
+Wrapping Slic in Macros
+------------------------
+
+The recommended way of integrating Slic into an application is to wrap the
+Slic API for logging messages into a set of convenience application macros that
+are used throughout the application code.
+
+This allows the application code to:
+
+* Centralize all use of Slic behind a thin macro layer,
+* Insulate the application from API changes in Slic,
+* Customize and augment the behavior of logging messages if needed, e.g.,
+  provide macros that are only active when the code is compiled with debug
+  symbols etc.
+
```

Comment:
Strongly recommend moving matter from "Slic Macros Used in Axom" section of appendix.rst into **a table**  (not a series of paragraphs) located here.  If I were considering adopting use of slic, I would feel much happier knowing that Axom already supplies macros for me to use, and additionally supplies a way for me to write my own.  If you lead me to read all about how to write my own logging macros and only afterward tell me "oh, we already did this", I feel like you want me to do extra work by not telling me the easy way first.

If you want to keep the matter in the appendix, duplicate it here in a table.  As a last alternative, please put a prominent link from here to the "Slic Macros Used in Axom" section of the appendix.

Reply:
Macros encapsulate application specific logic and decisions that although are suitable for Axom, may not be suitable for an application. Applications can use the SLIC macros we provide, but, typically will wrap the functionality in their own macro layer. This make it also easier to maintain, e.g., guard the application code from API changes in Slic etc.

I did try listing the macros in a table, but, it was hard to maintain for all the macros given the way paragraphs are rendered within cells of the table.

I think in conjunction with the sidebar navigation, readers can quickly see all the macros available and zoom in to the information they are interested.

I'll add a link from here to the appendix.

Reply:
The link was already there. I rephrased the text a bit and tried to clarify this part.